modules/exploits/windows/http/geutebrueck_gcore_x64_rce_bo.rb

```diff
@@ -0,0 +1,286 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'nokogiri'
+require 'open-uri'
+
+class MetasploitModule < Msf::Exploit::Remote
+    include Msf::Exploit::Remote::Tcp
+
+    Rank = NormalRanking
+
+    def initialize(info = {})
+        super(update_info(info,
+                          'Name' => 'Geutebrueck GCore - GCoreServer.exe Buffer Overflow RCE',
+                          'Description'	=> 'This module exploits a stack Buffer Overflow in the GCore server (GCoreServer.exe). The vulnerable webserver is running on Port 13003 and Port 13004, does not require authentication and affects all versions from 2003 till July 2016 (Version 1.4.YYYYY).',
+                          'License' => MSF_LICENSE,
+                          'Author' =>
+                          [
+                              'Luca Cappiello',
+                              'Maurice Popp'
+                          ],
+                          'References' =>
+                          [
+                              ['www.geutebrueck.com', '']
+                          ],
+                          'Platform' => 'win',
+                          'Targets' =>
+                          [
+                              ['Automatic Targeting', { 'auto' => true, 'Arch' => ARCH_X86_64 }],
+                              ['GCore 1.3.8.42, Windows x64 (Win7, Win8/8.1, Win2012R2,...)', { 'Arch' => ARCH_X86_64 }],
+                              ['GCore 1.4.2.37, Windows x64 (Win7, Win8/8.1, Win2012R2,...)', { 'Arch' => ARCH_X86_64 }]
+                          ],
+                          'Payload' =>
+                          {
+                              'Space' => '2000'
+                          },
+                          'Privileged' => false,
+                          'DisclosureDate' => '2017-01-24',
```

Comment:
Invalid date format.

Please run `./tools/dev/msftidy.rb modules/exploits/windows/http/geutebrueck_gcore_x64_rce_bo.rb` over the module.